### ISSUE TITLE
bars and lineWidth

### ISSUE THREAD

**Issue by GoogleCodeExporter** [josejaner/flot#353]
```

bars: {lineWidth: 0}

Safari, Chrome, Opera still display lines around bars.

Firefox and IE - ok.
```

Original issue reported on code.google.com by `yurk...@gmail.com` on 19 May 2010 at 10:53

Attachments:
- [gif.gif](https://storage.googleapis.com/google-code-attachments/flot/issue-353/comment-0/gif.gif)

**Comment by GoogleCodeExporter** [comment]
```
The problem was solved, take a look at the issue:
http://code.google.com/p/flot/issues/detail?id=253

The published flot 0.6 still does not have the patch for this problem, but HEAD 
of subversion has it. You can read the issue thread or download the last 
version from subversion.

Hope it helps.
```

Original comment by `jordi.ks...@gmail.com` on 16 Jul 2010 at 9:31

**Comment by GoogleCodeExporter** [comment]
Original comment by `olau%iol...@gtempaccount.com` on 13 Dec 2010 at 4:38
- Changed state: **Fixed**
